Dangerfile

```diff
@@ -0,0 +1,45 @@
+has_app_changes = !git.modified_files.grep(/lib/).empty?
+has_test_changes = !git.modified_files.grep(/spec/).empty?
+
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = pr_title.include? '#trivial'
+
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn('PR is classed as Work in Progress') if pr_title.include? '[WIP]'
+
+# Warn when there is a big PR
+warn('Big PR') if lines_of_code > 500
+
+# Add a CHANGELOG entry for app changes
+if !modified_files.include?('CHANGELOG.md') && has_app_changes
+  fail('Please include a CHANGELOG entry.')
+end
+
+# Warn about un-updated tests
+if has_app_changes && !has_test_changes
+  warn 'Tests were not updated'
+end
+
+if github.pr_body.length < 5
+  fail 'Please provide a summary in the Pull Request description'
+end
+
+# TODO: This could be a danger plugin
```

Comment:
Ah, just wrapping up tests 😉 At the rate it's going though, I'm sure I'll be making another one soon.